test/scraped_page_archive_test.rb

```diff
@@ -47,4 +47,19 @@
       end
     end
   end
+
+  describe '#git_url' do
+    subject { ScrapedPageArchive::GitStorage.new }
+
+    describe 'in a git repo where origin is an scp-style "URL"' do
+      it 'works as expected' do
+        with_tmp_dir do
+          remote_url = 'git@github.com:everypolitician-scrapers/random-scraper.git'
+          `git init`
+          `git remote add origin #{remote_url}`
```

Comment:
Personally I'd use `system` here since the shell invocation is unnecessary, and if `remote_url` were something different it might cause problems, but this is consistent with the style in the file here, so I think this is fine.